proposals/0185-vote-account-v4.md

```diff
@@ -184,13 +183,21 @@ other vote state versions, it's never stored with uninitialized state.
 
 #### `UpdateCommission`
 
-The existing `UpdateCommission` instruction will will continue to only update
-the inflation rewards commission in integer percentage values.
+The existing `UpdateCommission` instruction will continue to only update the
+inflation rewards commission in integer percentage values.
 
-When updating vote state v4 accounts, the new `inflation_rewards_commission_bps`
-field should be used instead of the old generic `commission` field.
-Additionally, the new commission value MUST be multiplied by `100` before being
-checked for commission or increases and before being stored in account data.
+The new commission value MUST be multiplied by `100` before being checked for
+commission or increases and before being stored in the new
+`inflation_rewards_commission_bps` field.
+
+#### `UpdateValidatorIdentity`
```

Comment:
I added this after realizing that the validator identity might get unexpectedly out of sync with the block revenue collector after the feature gate for this SIMD is activated because there will not yet be a way for nodes to update the block revenue collector account. This sync behavior should probably be removed in https://github.com/solana-foundation/solana-improvement-documents/pull/232, I will make amend that one too.